src/pages/result/index.tsx

```diff
@@ -36,44 +39,77 @@ const Result: NextPage = () => {
     selectedSortPriceValue,
     minPriceValue,
     maxPriceValue,
+    applyPrice,
     handleTradePeriodSelectChange,
     handleSortPriceSelectChange,
     handleMinPriceInputChange,
     handleMaxPriceInputChange,
     handlePriceApplyClick
   } = useSelectBoxFilter()
 
+  const categoryFilterOption =
+    selectedCategoryValue === 'ALL' ? null : selectedCategoryValue
+
+  const {
+    data: postList,
+    fetchNextPage,
+    hasNextPage,
+    refetch
+  } = useGetInfinitePostsQuery({
```

Comment:
```suggestion
  const infinitePosts = useGetInfinitePostsQuery({
```
여기도 객체로 받으면 좋을 것 같아요!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/174/commits/a6f681de442a40533851b770a8a71280628c236f#diff-33ad00554e4d8335fcb7f35a52e38907ad21791c810c9615fe473c46ff271abc)